hbase-asyncfs/src/test/java/org/apache/hadoop/hbase/security/HBaseKerberosUtils.java

```diff
@@ -43,7 +43,9 @@
   public static final String KRB_PRINCIPAL = SecurityConstants.REGIONSERVER_KRB_PRINCIPAL;
   public static final String MASTER_KRB_PRINCIPAL = SecurityConstants.MASTER_KRB_PRINCIPAL;
   public static final String KRB_KEYTAB_FILE = SecurityConstants.REGIONSERVER_KRB_KEYTAB_FILE;
+  @Deprecated
   public static final String CLIENT_PRINCIPAL = AuthUtil.HBASE_CLIENT_KERBEROS_PRINCIPAL;
+  public static final String CLIENT_PRINCIPAL_VALUE = AuthUtil.HBASE_CLIENT_KERBEROS_PRINCIPAL_VALUE;
```

Comment:
I think there's no need to deprecate `CLIENT_PRINCIPAL` and methods below, since it is a test class only. We can keep `CLIENT_PRINCIPAL` and replace the value with the correct one. Same to the methods.